openhtf/__init__.py

```diff
@@ -83,10 +87,22 @@ class TestPhaseInfo(mutablerecords.Record(
   """
 
   @classmethod
-  def WrapOrReturn(cls, func):
+  def WrapOrCopy(cls, func):
+    """Return a new TestPhaseInfo from the given function or instance.
+    
+    We want to return a new copy so that you can reuse a phase with different
+    options, plugs, measurements, etc.
+    
+    Args:
+      func: A phase function or TestPhaseInfo instance.
+    
+    Returns:
+      A new TestPhaseInfo object.
+    """
     if not isinstance(func, cls):
-      func = cls(func, inspect.getsource(func))
-    return func
+      return cls(func, inspect.getsource(func))
+    # We want to copy so that a phase can be reused
+    return copy.copy(func)
```

Comment:
yea i guess there's no cleaner way to do it.. if it were me, i'd make it a method on the record base class, next to __copy__ and __deepcopy__, and call it something clever that i can't think of right now, but here's some ideas... RecordCopy (like deepcopy, but different), CopyRecord, CopyFields (borrowing some nomenclature from namedtuple).  entirely your call, mutablerecords is your thing.